server/models/readingstat.js

```diff
@@ -0,0 +1,19 @@
+export default (sequelize, DataTypes) => {
+  const ReadingStat = sequelize.define('ReadingStat', {
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+    articleId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+    timeRead: {
```

Comment:
Whats the difference between timeRead and createdAt

Reply:
@seyi-adeleke the timeRead field is the field to be updated when the user re-reads the same article.